pulser-core/pulser/devices/__init__.py

```diff
@@ -27,4 +32,5 @@
     Chadoq2,
     IroiseMVP,
     AnalogDevice,
+    DigitalAnalogDevice,
```

Comment:
What is the use of _valid_devices ? I guess it's okay to keep Chadoq2 and IroiseMVP there, otherwise we could have put them in a list `_deprecated_devices` ?

Reply:
It's an internal parameter so I didn't bother. It's only used here: https://github.com/pasqal-io/Pulser/blob/b18e43428e9aa5f7c79010a3bacfc9dd3f5263ef/pulser-core/pulser/json/supported.py#L75